src/client/sandbox/node/element.js

```diff
@@ -349,12 +350,23 @@ export default class ElementSandbox extends SandboxBase {
 
             if (domUtils.isFileInput(el))
                 this.addFileInputInfo(el);
+
+            else if (domUtils.isBaseElement(el)) {
+                var storedHrefAttrName  = domProcessor.getStoredAttrName('href');
+                var storedHrefAttrValue = el.getAttribute(storedHrefAttrName);
+
+                urlResolver.updateBase(storedHrefAttrValue, this.document);
+            }
+
         }
     }
 
     onElementRemoved (el) {
         if (domUtils.isBodyElement(el))
             this.shadowUI.onBodyElementMutation();
+
+        else if (domUtils.isBaseElement(el))
```

Comment:
create the separate issue - https://github.com/DevExpress/testcafe-hammerhead/issues/385